Fix spelling of 'separate' in the man page and source code.
--- a/dcfldd.1
+++ b/dcfldd.1
@@ -68,7 +68,7 @@
 send MD5 hash output to FILE instead of stderr
 .IP
 if you are using multiple hash algorithms you
-can send each to a seperate file using the
+can send each to a separate file using the
 convention ALGORITHMlog=FILE, for example
 md5log=FILE1, sha1log=FILE2, etc.
 .TP
--- a/dcfldd.c
+++ b/dcfldd.c
@@ -181,7 +181,7 @@
                              in a comma separated list\n\
   hashlog=FILE             send MD5 hash output to FILE instead of stderr\n\
                              if you are using multiple hash algorithms you\n\
-                             can send each to a seperate file using the\n\
+                             can send each to a separate file using the\n\
                              convention ALGORITHMlog=FILE, for example\n\
                              md5log=FILE1, sha1log=FILE2, etc.\n\
   hashlog:=COMMAND         exec and write hashlog to process COMMAND\n\
